shpc/client/config.py

```diff
@@ -41,7 +41,8 @@ def main(args, parser, extra, subparser):
                     % param
                 )
                 continue
-            key, value = param.split(":", 1)
+            value, key = param[::-1].split(":", 1)
```

Comment:
Can you walk me through this logic? It doesn't read clearly for the developer reader to get a sense of what is going on. Perhaps add an example to the comment?

Reply:
Sure, let me explain, and maybe we'll find out there's a neater way to code this.
The old split in line 44 will split by a single colon entry, the first one encountered in the string, to produce a key,value pair.
What we need for the fix is to split by the LAST colon entry. 
So I thought, let's revert the string, split by the first colon entry, and then revert back:
```
# split reversed string by first colon entry, which is the last entry in the original string
value, key = param[::-1].split(":", 1)
# reverse back the (key, value) strings
key, value = key[::-1], value[::-1]
```

Does it make sense?
Do you think there's a cleaner way to achieve this?

Thanks for your feedback! :)